software/contrib/README.md

```diff
@@ -39,9 +39,15 @@ A semi-random LFO that uses [John Conway's Game Of Life](https://en.wikipedia.or
 Record 6 banks of 6 channels of control voltage, and then play them back at a consistent rate set by an external clock.
 Recording of CV can be primed so that you can record a movement without missing a beat
 
-<i>Author: [anselln](https://github.com/anselln)</i>
+<i>Author: [gamecat69](https://github.com/gamecat69)</i>
```

Comment:
[question]
Did you mean to change the author of the CVecorder script? Or was this an accidental copy & paste from below?